database/migrations/2024_05_19_154057_recreate_settings.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use App\Models\Setting;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::dropIfExists('settings');
```

Comment:
Also, don't just drop all old settings. Migrate them to the new format.

Reply:
Yes the old settings will be here to. I am not done yet.